Documentation/_kconfig/helper.jinja

```diff
@@ -0,0 +1,56 @@
+{% macro help(sc) -%}
+Help
+====
+{% for node in sc.nodes -%}
+{% if node.help %}
+
+.. parsed-literal::
+
+    {{node.help|indent(4)}}
+{% endif %}
+{% endfor %}
+{%- endmacro %}
+{%- macro direct_dep(sc) -%}
+{% if sc.direct_dep == sc.kconfig.y %}
+{% else %}
+Direct Dependencies
+===================
+
+{{expr_str(sc.direct_dep)}}
+{% endif %}
+{%- endmacro %}
+{%- macro defaults(sc, choice) -%}
+Defaults
+========
+{% if sc.defaults %}
+{%- for value, cond in sc.orig_defaults %}
+- {{expr_str(value)}}{% if cond != sc.kconfig.y %} if {{expr_str(cond)}}{% endif %}
+{%- endfor %}
+{% else %}
+No defaults defined. Implicitly defined to {% if choice %}the first (visible) choice option.{% elif sc.orig_type in [kconfiglib.BOOL, kconfiglib.TRISTATE] %}``n``.{% else %}the empty string.{% endif %}
+{% endif %}
+{%- endmacro %}
+{%- macro kconfig_definition(sc) -%}
+Kconfig Definitions
+===================
+
+.. highlight:: kconfig
+
+{% for node in sc.nodes %}
+
+At ``{{node.filename}}:{{node.linenr}}``
+{% if node.include_path %}
+Included via: {% for filename, linenr in node.include_path %}``{{filename}}:{{linenr}}``{% if not loop.last %} -> {% endif %}{% endfor %}
```

Comment:
We could probably use :file: for filenames and later generate links to GitHub (to master or appropriate tag).

Reply:
I like this idea. Not quite sure how we would handle this cross repo, but something to think about.